libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -128,3 +131,38 @@ void StaticAnalyzer::warning(SourceLocation const& _location, string const& _des
 
 	m_errors.push_back(err);
 }
+
+bool StaticAnalyzer::visit(InlineAssembly const& _inlineAssembly)
+{
+	if (m_currentFunction)
```

Comment:
Not sure which is more readable, but alternatively `if (!m_currentFunction) return true` would reduce the indentation one level.